test/spec/Feature/Query/EmbedDisambiguationSpec.hs

```diff
@@ -103,22 +103,22 @@ spec =
               "details": [
                 {
                   "cardinality": "many-to-many",
-                  "relationship": "test.whatev_jobs[whatev_jobs_site_id_1_fkey][whatev_jobs_project_id_1_fkey]",
+                  "relationship": "whatev_jobs using whatev_jobs_site_id_1_fkey and whatev_jobs_project_id_1_fkey",
```

Comment:
Can we add the fk columns for the m2m cases too? That would make the error more complete. It would look something like:

```json
"relationship": "whatev_jobs using whatev_jobs_site_id_1_fkey(job_id) and whatev_jobs_project_id_1_fkey(site_id)"
```

Reply:
Just to confirm, the columns will be the ones in the intermediate table, right? In this case the test would be:

```
"relationship": "whatev_jobs using whatev_jobs_site_id_1_fkey(site_id_1) and whatev_jobs_project_id_1_fkey(project_id_1)"
```

Reply:
Yes, correct!